packages/base-driver/test/e2e/express/server.e2e.spec.js

```diff
@@ -1,16 +1,40 @@
-import {server} from '../../../lib';
+import {SslHandler, server} from '../../../lib';
 import axios from 'axios';
 import {createSandbox} from 'sinon';
 import B from 'bluebird';
 import _ from 'lodash';
 import {TEST_HOST, getTestPort} from '@appium/driver-test-support';
+import {given} from 'mocha-testdata';
+import {promisify} from 'util';
+
+const exec = promisify(require('child_process').exec);
```

Comment:
in the appium project we typically use the `teen_process` library instead of `child_process`. lots of examples around of how exec works with that.